frontend/src/metabase/static-viz/components/XYChart/Values/Values.tsx

```diff
@@ -68,8 +68,8 @@ export default function Values({
   if (containBars && xScale.bandwidth) {
     const innerBarScaleDomain = multipleSeries
       .filter(series => series.type === "bar")
-      .map((barSerie, index) => {
-        barSeriesIndexMap.set(barSerie, index);
+      .map((barSeries, index) => {
```

Comment:
Rename `serie` to `series` as `serie` is grammatically incorrect, and we solve the ambiguity of multiple series by using the name `multipleSesies` and `series` refers to a single series instead.